plugins/com.google.cloud.tools.eclipse.projectselector/META-INF/MANIFEST.MF

```diff
@@ -8,3 +8,33 @@ Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
+Export-Package: com.google.cloud.tools.eclipse.projectselector
+Import-Package: com.google.api.client.auth.oauth2,
+ com.google.api.client.googleapis,
+ com.google.api.client.googleapis.services,
+ com.google.api.client.googleapis.services.json,
+ com.google.api.client.http,
+ com.google.api.client.http.javanet,
+ com.google.api.client.json,
+ com.google.api.client.json.jackson,
+ com.google.api.client.util,
+ com.google.cloud.tools.eclipse.login.ui,
+ com.google.cloud.tools.eclipse.util,
+ com.google.cloud.tools.eclipse.util.status,
+ com.google.common.annotations;version="20.0.0",
+ com.google.common.base;version="20.0.0",
+ com.google.common.collect;version="20.0.0",
```

Comment:
These ranges need an upper bound, so "[20.0.0,21.0.0)", as otherwise we could pull in Guava 21, which has breaking APIs.